docs/releaseNotes.yml

```diff
@@ -35,7 +35,12 @@ items:
     prevVersion: 2.1.0
     date: 'TBD'
     notes:
-      # next changelog item here
+      - title: Docker BuildKit always used for builds
+        type: change
+        body: >-
+          Docker BuildKit is enabled for all Emissary builds. Additionally, the Go
+          build cache is fully enabled when building images, speeding up repeated builds.
+        docs: https://docs.docker.com/develop/develop-images/build_enhancements/
```

Comment:
We don't usually put tooling changes like this in the changelog, do we?

Reply:
We did when we first forced `DOCKER_BUILDKIT` to 0 in the Makefile (since it was affecting people trying to build) so it seemed like we should include this one.